src/views/getProfile.js

```diff
@@ -12,15 +12,14 @@ export class Profile extends Component {
   }
 
   render() {
-    const { fetched } = this.props;
-    if (!fetched) {
+    const { isFetching } = this.props;
+    if (isFetching === true) {
```

Comment:
If you just say, isfetching, then it will definately translate to true. No need for ```== true```